src/components/Input.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const InputScreen = () => {
+  const submitInput = (e) => {
+    e.preventDefault();
+    // setTodo({ id: uuidv4, title: e.target.value, completed: false });
```

Comment:
Please kindly remove the unused code. Comments that are not needed can clutter up the code and make it more difficult to read and understand

Reply:
Thanks, will make the required changes on this.